content/en/profiler/enabling/go.md

```diff
@@ -22,7 +22,9 @@ The profiler is shipped within Datadog tracing libraries. If you are already usi
 
 ## Requirements
 
-The Datadog Profiler requires Go 1.12+.
+The Datadog Profiler requires Go 1.12+. For Endpoints profiling.
+
+For Code Hotspots and Endpoints profiling, use Go version 1.18 or greater and `dd-trace-go` version 1.37.0 or greater.
```

Comment:
from here, it looks like Endpoints requires Go 1.12+ and dd-trace-go 1.37.0+, and Code Hotspots requires Go 1.18+ and dd-trace-go 1.37.0+ ..... is there a way to combine the information in lines 25-27 for more clarity?